packages/altive_lints/pubspec.yaml

```diff
@@ -15,6 +15,6 @@ environment:
   sdk: ^3.0.0
 
 dependencies:
-  analyzer: ^6.5.0
+  analyzer: ^6.7.0
   collection: ^1.18.0
-  custom_lint_builder: ^0.6.5
+  custom_lint_builder: ^0.7.0
```

Comment:
In `custom_lint` and `custom_lint_builder` 0.7.0, compatibility has been expanded to support multiple versions of the `analyzer`!

Up until the previous version, 0.6.10, `analyzer` was specified as ^6.10.0, which prevented projects using Flutter Stable from upgrading the custom_lint version. However, with this latest update, the newest version is now available for use on Stable as well.